bitrepository-core/src/main/java/org/bitrepository/common/utils/Base16Utils.java

```diff
@@ -58,7 +58,8 @@ public static byte[] encodeBase16(String hexString) throws DecoderException {
         try {
             return Hex.decodeHex(hexString);
         } catch (DecoderException e) {
-            throw new DecoderException("The string '" + hexString + "' is not a valid hex-string.");
+            throw new DecoderException(
```

Comment:
It’s ugly that our own class throws an Apache exception type. I didn’t find the time ripe for changing it now, but I will want to change it at a point.